### DISCUSSION TITLE
Detect objects that stay in a zone (and not just passing)

### DISCUSSION THREAD

**Discussion by mama31** [blakeblackshear/frigate#3712] [Ask A Question]
Hello,
Is there a way to set frigate to detect objects that enter and stay in a zone for more than x seconds (and are not just passing through).
it is a little the opposite of avoiding stationary objects.
Basically I want to detect cars that park in front of my house and not just passing in front.
Thanks!

**Comment by NickM-27** [comment]
You can do this using the MQTT events https://docs.frigate.video/integrations/mqtt#frigateevents in home assistant or otherwise, but there's nothing specific built into frigate
